src/zkevm_specs/evm/typing.py

```diff
@@ -55,8 +67,7 @@ class Transaction:
     id: int
     nonce: U64
     gas: U64
-    gas_tip_cap: U256
-    gas_fee_cap: U256
+    gas_price: U256
```

Comment:
does `gas_price` really need `U256` size ?

Reply:
I am not sure actually, I basically tried to follow those types used in `geth`. Some of them like `nonce` and `gas` have less bits than the one specified in yellow paper, but others match.

So I was thinking there might be some reason for `geth` to have `big.Int` for `gas_price`, so I keep it as `U256` here.

Reply:
if so,  does `N_BYTES_GAS = 8` of param.py need to change accordingly ?

Reply:
`N_BYTES_GAS` is for `gas` itself, tho it could have value up to 256 bits in yellow paper, but it's used as `uint64` in `geth`, so `N_BYTES_GAS = 8`.

While `gas_price` is the unit that maps `gas` to real `ether`, which also could have value up to 256 bits, and used as `big.Int` in `geth`, so we try not to change it in case there is some edge cases that haven't came to our minds.

Reply:
OK to me for now! Thx